pom.xml

```diff
@@ -116,7 +116,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>folio-kafka-wrapper</artifactId>
-      <version>2.0.5</version>
+      <version>2.4.0-SNAPSHOT</version>
```

Comment:
Please don't introduce another SNAPSHOT dependency until after this module is released for 2021 R2, as SNAPSHOT dependencies stop the module from being released.

Reply:
mod-inventory has been released so this change can continue